configs/brh_config.json

```diff
@@ -125,6 +125,27 @@
 				"commons": "Kids First Data Resource Center",
 				"study_url": "path:link"
 			}
+		},
```

Comment:
remove this file and the config directory and put it into docs as a sample config.

Reply:
Just did this myself